s29862-JuliaButkiewicz/Zadania_6-7/Zadania_6-7

```diff
@@ -0,0 +1,74 @@
+Poniżej znajduje się sprawozdanie z wykonywanych zadań z PowerShella.
+---------------------------------------------------------------------
+
+Dla pewności, od razu uruchomiłam powłokę Microsofto PowerShell z poziomu
+administratora by mieć pewność, że każda część skryptu będzie możliwa do
```

Comment:
Rozumiem, że to tyczy się tylko samego początku a nie całości skryptu.
Jedynym poleceniem, które trzeba było wywołać jako administrator to `Set-ExecutionPolicy Bypass` a cała reszta miała być wywoływana z terminala bez uprawnień administratora.